### PR TITLE
[mcs/tests] add interp target

### PR DESCRIPTION
**Pull request by lewurm**

### CODE REVIEW

**Comment by lewurm** [comment]
@monojenkins build interpreter

**Comment by lewurm** [comment]
build interpreter

**Comment by lewurm** [comment]
@monojenkins build interpreter

File: mcs/tests/Makefile [hunk 1]
```diff
@@ -70,10 +71,15 @@ qcheck2:
...
 	$(TESTER) -mode:nunit -files:'v2' -compiler:$(COMPILER) -issues:known-issues-mt -compiler-options:"-lib:$(topdir)/class/lib/monotouch projects/MonoTouch/ivt.cs"
 
+check-interp: compile-all setup build-compiler-lib
+	$(TESTER_INTERP) -mode:pos -files:$(TEST_PATTERN) -compiler:$(COMPILER) -issues:known-issues-interp-$(PROFILE) -log:$(PROFILE).log -il:ver-il-$(PROFILE).xml $(DEFINES) $(TOPTIONS)
+
 test-local:
 	@:
 
-run-test-local: $(TEST_ILS:.il=.dll) setup qcheck
+compile-all: $(TEST_IL:.il=.dll)
```
Comment (marek-safar) [inline mcs/tests/Makefile]: You have a typo here, it needs to be TEST_ILS

**Comment by lewurm** [comment]
build interpreter

**Comment by lewurm** [comment]
build interpreter

**Comment by lewurm** [comment]
build interpreter

**Comment by lewurm** [comment]
build interpreter

**Comment by lewurm** [comment]
build interpreter

**Comment by lewurm** [comment]
build interpreter

**Comment by lewurm** [comment]
@vargaz this is ready for review, please take a look

**Comment by lewurm** [comment]
for the curious reviewer, on our Linux x64 machines:

- running mcs/tests with the JIT: 160s
- running mcs/tests with the interpreter: 2393s

so the interpreter is factor ~15x slower.

**Comment by lewurm** [comment]
@monojenkins build OS X x64

**Comment by lewurm** [comment]
@monojenkins build OS X i386

**Comment by lewurm** [comment]
@monojenkins build Linux AArch64 Interpreter

**Comment by lewurm** [comment]
@monojenkins build Linux AArch64

**Comment by lewurm** [comment]
@monojenkins build Linux ARMv5 soft float

**Comment by lewurm** [comment]
@monojenkins build Linux ARMv7 hard float

**Comment by lewurm** [comment]
@monojenkins build Linux ARMv7 hard float Interpreter

**Comment by lewurm** [comment]
@marek-safar @akoeplinger are you okay with the `Makefile` changes?

**Pull request merged**